io_scene_niftools/modules/nif_export/animation/transform.py

```diff
@@ -239,31 +239,16 @@ def export_transforms(self, parent_block, b_obj, b_action, bone=None):
             scale_curve.append((frame, scale[0]))
 
         if n_kfi:
-            if max(len(c) for c in (quat_curve, euler_curve, trans_curve, scale_curve)) > 1:
-                # number of frames is > 1, so add transform data
+            # set the default transforms of the interpolator as the bone's bind pose
+            n_kfi.translation.x, n_kfi.translation.y, n_kfi.translation.z = bind_trans
+            n_kfi.rotation.w, n_kfi.rotation.x, n_kfi.rotation.y, n_kfi.rotation.z = bind_rot.to_quaternion()
+            n_kfi.scale = bind_scale
+
+            if max(len(c) for c in (quat_curve, euler_curve, trans_curve, scale_curve)) > 0:
+                # number of frames is > 0, so add transform data
                 n_kfd = block_store.create_block("NiTransformData", exp_fcurves)
                 n_kfi.data = n_kfd
             else:
-                # only add data if number of keys is > 1
```

Comment:
Is this no longer true, or was that based on wrong information?

Reply:
I'm pretty sure that was never the correct behavior, at least from what kormakur was describing.